test/integration/function-basic/index.spec.ts

```diff
@@ -31,3 +32,33 @@ test('functions basics', async ({ page, api }) => {
   await page.locator('text="echo, parameter: bound foo parameter"').waitFor({ state: 'visible' });
   await page.locator('text="echo, secret: Some bar secret"').waitFor({ state: 'visible' });
 });
+
+test('function editor save shortcut', async ({ page, api, browserName }) => {
+  const dom = await readJsonFile(path.resolve(__dirname, './functionDom.json'));
+
+  const app = await api.mutation.createApp(`App ${generateId()}`, {
+    from: { kind: 'dom', dom },
+  });
+
+  const editorModel = new ToolpadEditor(page, browserName);
+  await editorModel.goto(app.id);
+
+  await expect(editorModel.appCanvas.getByText('edited hello')).toBeVisible();
+
+  await editorModel.componentEditor.getByRole('button', { name: 'edited' }).click();
```

Comment:
When we add more tests for the query editor it might make sense for some of these to have methods we can reuse.